types/chrome/index.d.ts

```diff
@@ -6242,6 +6242,10 @@ declare namespace chrome.pageCapture {
      * Parameter mhtmlData: The MHTML data as a Blob.
      */
     export function saveAsMHTML(details: SaveDetails, callback: (mhtmlData?: Blob) => void): void;
+    /**
+     * Promises are supported in Manifest V3 and later, but callbacks are provided for backward compatibility. You cannot use both on the same function call. The promise resolves with the same type that is passed to the callback.
```

Comment:
IMO the comment isn't necessary since the types describe this - This [is common for all (AFAICT) callback -> Promise methods](https://developer.chrome.com/s/results?q=%22callbacks%20are%20provided%20for%20backward%20compatibility%22#gsc.tab=0&gsc.q=%22callbacks%20are%20provided%20for%20backward%20compatibility%22&gsc.sort=). Instead it'd be useful to mark the type as requiring MV3. 

```suggestion
     * Saves the content of the tab with given id as MHTML.
     * @since Chrome 116 MV3
```

Reply:
Thanks for the review! Landed in [c5318e4](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68100/commits/c5318e40a347d01918e4521fbb6c0064046ee169)